lib/hooks/userspace/hook-express.js

```diff
@@ -98,12 +98,12 @@ function startRootSpanForRequest(req) {
  */
 function endRootSpanForRequest(rootContext, req, res) {
   if (req.route && req.route.path) {
-    rootContext.span.setLabel(
+    rootContext.addLabel(
       'express/request.route.path', req.route.path);
   }
-  rootContext.span.setLabel(
+  rootContext.addLabel(
       TraceLabels.HTTP_RESPONSE_CODE_LABEL_KEY, res.statusCode);
-  rootContext.close();
+  rootContext.spanData_.close();
```

Comment:
Yea, there are private accesses all over due to the extra opaque span wrapping. I was hoping to clean this out when I delete `OpaueSpan` next but if it needs to go now I can do that as part of this PR.